app/dataSources/dbFetch.tsx

```diff
@@ -126,3 +116,40 @@ export async function writeGoogleUserData (googleUserId: string){
     await redis.json.set(`user:${googleUserId}`, "$", leagueConfigString)
 }
 
+export async function getAllKeys(keyPrefix: string): Promise<string[]> {
+
+    const redis = new Redis({
+        url: process.env.KV_REST_API_URL,
+        token: process.env.KV_REST_API_TOKEN
+    });
+
+    let allKeysCursor = await redis.scan("0", {match: keyPrefix});
+    let allKeysResults = allKeysCursor[1];
+    let cursorStart = allKeysCursor[0];
+    while(cursorStart !== "0"){
+        allKeysCursor = await redis.scan(cursorStart, {match: keyPrefix});
+        allKeysResults = allKeysResults.concat(allKeysCursor[1]);
+        cursorStart = allKeysCursor[0];
+    }
+
+    return allKeysResults;
+}
+
+export async function getJson<T>(key: string): Promise<T> {
```

Comment:
What is type `T`?

Reply:
It is whatever type you want it to be 😀

Reply:
Answered: https://github.com/microsoft/TypeScript/issues/19244

Reply:
😒